### PR TITLE
Add insecure wss connection

### PR DESCRIPTION
**Pull request by Demacr**
In case of self-signed certificates.

### CODE REVIEW

**Comment by ThomasHabets** [comment]
@googlebot ping?

**Comment by Demacr** [comment]
@ThomasHabets I signed CLA last time.

**Comment by ThomasHabets** [comment]
Oh yeah. Still, I expected googlebot to come in on demand and say so. Maybe I have to create the labels. I just did, so maybe next time googlebot will be happy.

File: huproxyclient/client.go [hunk 1]
```diff
@@ -85,7 +87,15 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	dialer := websocket.Dialer{}
+	var tlsConfig *tls.Config
+	if *insecure {
+		tlsConfig = &tls.Config{InsecureSkipVerify: true}
+	} else {
+		tlsConfig = nil
```
Comment (ThomasHabets) [inline huproxyclient/client.go]: This is a no-op.

Simpler code:

```go
dialer := websocket.Dialer{}
if *insecure {
  dialer.TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
}
```

Comment (Demacr) [reply huproxyclient/client.go]: Fixed

**Comment by ThomasHabets** [comment]
Thanks!

**Pull request merged**